Framework/MDAlgorithms/src/ConvToMDEventsWS.cpp

```diff
@@ -47,11 +47,7 @@ size_t ConvToMDEventsWS::convertEventList(size_t workspaceIndex) {
   const typename std::vector<T> &events = *events_ptr;
 
   // Iterators to start/end
-  typename std::vector<T>::const_iterator it = events.begin();
-  typename std::vector<T>::const_iterator it_end = events.end();
-
-  it = events.begin();
-  for (; it != it_end; it++) {
+  for (auto it = events.cbegin(); it != events.cend(); it++) {
```

Comment:
Note to self: This would change behavior, in case events get modified during loop (by other thread?)?